scripts/local-install.sh

```diff
@@ -53,14 +48,16 @@ COMPRESSED_PACKAGE="$(tail -1 /tmp/pack.out)"
 tar -xzf "${COMPRESSED_PACKAGE}"
 rm "${COMPRESSED_PACKAGE}"
 mv package "${PACKAGE_DIR}"
-PACKAGE_NAME=$(cat ./package.json | jq -r '.name')
+PACKAGE_NAME=$(node -p "require('./package.json').name")
 popd > /dev/null
 
 ### Install local package to target package
 pushd "$2" > /dev/null
 TARGET_DIR="$(pwd)"
 rm -rf node_modules/.cache
-yarn remove "${PACKAGE_NAME}"
+if [ "$(grep -c '"'"$PACKAGE_NAME"'"' ./package.json)" -gt 0 ]; then
+  yarn remove "${PACKAGE_NAME}"
+fi
```

Comment:
This makes sure that `yarn remove` is only run when the package being installed already exists. Previously, if you were installing `@eppo/js-client-sdk` in a freshly generated React app, this script would fail.